doc/getting_started/index.rst

```diff
@@ -159,6 +159,12 @@ Set Up a Toolchain
    setting :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` appropriately for a non-SDK
    toolchain.
 
+   As another way to skip setting up a toolchain, you may be able for
```

Comment:
Hm, I think what's being skipped is the step, not setting up a toolchain. Using the SDK means you have already set up a toolchain, so the step in this point in the docs can be skipped because it is already done.

How about this:

```
On some configurations (e.g. non-MCU x86 and x86_64 boards on Linux), you
can also skip this step by using the toolchain provided by your host operating system.
```

Reply:
This (tested) suggestion is about literally not setting up any toolchain or SDK... just "apt-get/dnf install gcc" and roll.

Reply:
I purposely avoided "toolchain" for this case because (correctly or not) "toolchain" is never how users refer to their "host compiler". So mentioning a toolchain could confuse people into thinking they still to install some kind of "toolchain" and defeat the whole point which is to save them doing anything.